projects/laji/src/app/shared-modules/spreadsheet/service/mapping.service.ts

```diff
@@ -365,7 +366,7 @@ export class MappingService {
   }
 
   mapKeywords(value) {
-    return value.split(new RegExp([...MappingService.valueSplitter, ','].join('|'), 'g')).map(val => val.trim());
+    return value.split(new RegExp(MappingService.keywordSplitters.join('|'), 'g')).map(val => val.trim());
```

Comment:
@vixriihi Related to what you said here "If you test this with keyword value k1;k2,k3. The current code will will only save ['k1'] value to keywords and drops the k2 and k3 values all together." Now i changed and i don't see that mistake